validators/src/unit-test/java/uk/gov/moj/sdt/validators/BulkSubmissionValidatorTest.java

```diff
@@ -279,25 +274,22 @@ public void testDuplicateIndividualRequest() {
         errorMessage.setErrorCode(IErrorMessage.ErrorCode.DUPLD_CUST_REQID.name());
         errorMessage.setErrorText("Unique Request Identifier has been specified more "
                 + "than once within the originating Bulk Request.");
-        errorMessagesCache = EasyMock.createMock(ICacheable.class);
-        expect(errorMessagesCache.getValue(IErrorMessage.class, IErrorMessage.ErrorCode.DUPLD_CUST_REQID.name()))
-                .andReturn(errorMessage);
-        replay(errorMessagesCache);
+        when(errorMessagesCache.getValue(IErrorMessage.class, IErrorMessage.ErrorCode.DUPLD_CUST_REQID.name()))
+                .thenReturn(errorMessage);
         validator.setErrorMessagesCache(errorMessagesCache);
 
         bulkSubmission.accept(validator, null);
 
         // Check the duplicate individual request has been rejected
-        Assert.assertEquals(IIndividualRequest.IndividualRequestStatus.REJECTED.getStatus(), bulkSubmission
+        Assertions.assertEquals(IIndividualRequest.IndividualRequestStatus.REJECTED.getStatus(), bulkSubmission
```

Comment:
Add a static import of Assertions.fail() to the class so you can drop the "Assertions." from this line. This will make it consistent with the other classes.